src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceHelpers.cs

```diff
@@ -20,6 +22,9 @@ public static class WindowsServiceHelpers
         /// </returns>
         [SupportedOSPlatformGuard("windows")]
         public static bool IsWindowsService()
+            => _isWindowService ??= GetIsWindowsService();
```

Comment:
The initialization doesn't need to be "lazy" in this way. It can be in static field initializer and will run when this helper class is first touched.

Reply:
reasonable, thanks

Reply:
I use this way to follow the style in `SystemdHelper` 

https://github.com/dotnet/runtime/blob/111b089775820886b01607f4c7d0f6a2e7b81226/src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdHelpers.cs#L27-L28

Should I update the usage in `SystemdHelper`?